src/Migrations/Version20191030201837.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ilios\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Increases learning material title column size.
+ */
+final class Version20191030201837 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Increases learning material title column size.';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function up(Schema $schema) : void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE learning_material CHANGE title title VARCHAR(120) NOT NULL');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function down(Schema $schema) : void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE learning_material CHANGE title title VARCHAR(60) NOT NULL COLLATE utf8_unicode_ci');
```

Comment:
confused why this has `COLLATE` one way and not the other.

Reply:
dunno. i'm gonna lob it off.